control/tests/statesp_test.py

```diff
@@ -591,12 +594,19 @@ def test_pow(self, sys222, sys322):
                 ss2tf(expected).minreal(),
                 ss2tf(result).minreal(),
             )
-            result = (sys**-1 * sys).minreal()
-            expected = StateSpace([], [], [], np.eye(2), dt=0)
-            assert _tf_close_coeff(
-                ss2tf(expected).minreal(),
-                ss2tf(result).minreal(),
-            )
+            try:
+                result = (sys**-1 * sys).minreal()
+                expected = StateSpace([], [], [], np.eye(2), dt=0)
+                assert _tf_close_coeff(
+                    ss2tf(expected).minreal(),
+                    ss2tf(result).minreal(),
+                )
+            except AssertionError:
+                if platform.system() == 'Darwin':
+                    pytest.xfail("minreal bug on MacOS")
+                else:
+                    raise
```

Comment:
See also 2359299f34f23147dc9417481d2103cb9e048af5 in #1109

Reply:
I'll plan to merge #1109 first, then rebase and remove this exception.